.github/workflows/ci_pr.yaml

```diff
@@ -47,6 +47,9 @@ jobs:
       timeout-minutes: 20
       run: |
         golint ./... > golint.err 2>&1
+        if ! [ -s golint.err ]; then
```

Comment:
I think it's worth adding a ci-main workflow that runs on commits to the `main` branch, in addition to this PR workflow. the workflow can look identical.

Reply:
Yup. That's true. Ive added it in https://github.com/GoogleCloudPlatform/anthos-samples/pull/26/commits/4cdd344e7f7e9e387c38fc83d70ba0e974e6890a